### PR TITLE
Remove react-intl.

### PR DESCRIPTION
**Pull request by sri2k1us**
- SSIA

### CODE REVIEW

**Comment by ianmcorvidae** [comment]
At least as far as bundle size, looks good. 258kB first load JS to 245kB, and I don't see react-intl anywhere (presumably what's left in ui-lib is getting tree-shaken away.

File: src/components/analyses/details/InfoPanel.js [hunk 1]
```diff
@@ -142,6 +143,8 @@ function Updates(props) {
                 columnData={columns}
                 rowsInPage={updates.length}
                 baseId={baseId}
+                selectAllLabel={i18nCommon("selectAllLabel")}
+                sortLabel={i18nCommon("sortLabel")}
             />
```
Comment (sri2k1us) [inline src/components/analyses/details/InfoPanel.js:148]: May be we could have another component in `Sonora` that will pass `selectAllLabel` and `sortLabel` prop by default to `EnhancedTableHead`.

Comment (ianmcorvidae) [reply src/components/analyses/details/InfoPanel.js:148]: That would make sense to me, to save some repetition. I imagine it'd pretty much just be `export default (props) => <EnhancedTableHead selectAllLabel={t("selectAllLabel")} sortLabel={t("sortLabel")} {...props} />`, easily thrown into a file somewhere or other.

**Review by ash3rz — APPROVED** [review]
LGTM, thanks for tackling this 🎉  :shipit: 
One thing I didn't see was any page updates for `namespacesRequired` - not sure if those pages already had everything set up or not.
Also not sure if there should also be a `package-lock.json` update with this?  Not important, just figured if we removed a dependency that it would have updates.

**Comment by ianmcorvidae** [comment]
It looks like it probably should change package-lock, yeah. I think running `npm uninstall react-intl` will update the file appropriately.

**Comment by sri2k1us** [comment]
> LGTM, thanks for tackling this 🎉 :shipit:
> One thing I didn't see was any page updates for `namespacesRequired` - not sure if those pages already had everything set up or not.
> Also not sure if there should also be a `package-lock.json` update with this? Not important, just figured if we removed a dependency that it would have updates.

Good point. Let me update the lock file. As far as pages are concerned, I haven't checked the individual pages. But should be able track them with warnings and fix it easily.

**Comment by sri2k1us** [comment]
Thanks for reviews.

**Pull request merged**